devtools/forcefield-scripts/amber/conversion_script.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import print_function
+try:
+    from cStringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
Since you're importing ParmEd anyway, why not just do something like

```Python
from parmed.utils.six.moves import StringIO
```

and let the `six` library handle it?